styles.css

```diff
@@ -68,24 +99,23 @@ body {
 .columnRight label {
     display: none;
 }
+
 form .input-box input {
+    position: relative;
     height: 100%;
     width: 100%;
     outline: none;
-    border: 1.5px solid;
+    border: 1px solid;
     border-radius: 5px;
     font-size: 17px;
     transition: all 0.3s ease;
     padding: 5px;
-    border-color: rgb(199, 199, 199);  
+    color: rgb(163, 157, 157);
```

Comment:
What is the difference between this input box and the error class mentioned above?